src/test/frontend/handler/repo_test.cljs

```diff
@@ -1,18 +1,52 @@
 (ns frontend.handler.repo-test
   (:require [cljs.test :refer [deftest use-fixtures]]
             [frontend.handler.repo :as repo-handler]
+            [frontend.handler.page :as page-handler]
+            [frontend.handler.conversion :as conversion-handler]
             [frontend.test.helper :as test-helper]
+            [logseq.graph-parser.util :as gp-util]
             [logseq.graph-parser.cli :as gp-cli]
             [logseq.graph-parser.test.docs-graph-helper :as docs-graph-helper]
             [frontend.db.conn :as conn]))
 
 (use-fixtures :each {:before test-helper/start-test-db!
                      :after test-helper/destroy-test-db!})
 
+(defn- conversion-dir-ver3
+  [path]
+  (let [original-body (gp-util/path->file-body path)
+        ;; only test file name parsing, don't consider title prop overriding
+        rename-target (conversion-handler/calc-dir-ver-3-rename-target original-body nil)]
+    (if rename-target
+      (do (prn "conversion dir-ver3: " original-body " -> " rename-target)
+        (#'page-handler/compute-new-file-path path rename-target))
+      path)))
+
+(defn- convert-graph-files-path
+  "Given a list of files, converts them according to the given conversion function"
+  [files conversion-fn]
+  (map (fn [file]
+         (assoc file :file/path (conversion-fn (:file/path file)))) files))
+
 ;; Integration test that test parsing a large graph like docs
+;; Check if file name conversion from old version of docs is working
+(deftest ^:integration parse-and-load-v067-files-to-db
```

Comment:
Great to see an integration test for conversion :). I found the test name confusing as the test is about file name conversion. Maybe include file-name-conversion in the name?